test-configs.yaml

```diff
@@ -831,35 +815,53 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
-  qemu_arm:
+  qemu_i386:
     name: qemu
     mach: qemu
-    arch: arm
+    arch: i386
     boot_method: qemu
+    context: {arch: 'i386', cpu: 'qemu32', guestfs_interface: 'virtio'}
     filters:
-      - whitelist: {defconfig: ['multi_v7_defconfig', 'vexpress_defconfig']}
+      - whitelist: {defconfig: ['i386_defconfig']}
 
-  qemu_arm64:
+  qemu_x86_64:
     name: qemu
     mach: qemu
-    arch: arm64
+    arch: x86_64
     boot_method: qemu
+    context: {arch: 'x86_64', cpu: 'qemu64', guestfs_interface: 'virtio'}
     filters:
       - whitelist: {defconfig: ['defconfig']}
 
-  qemu_i386:
-    name: qemu
+  qemu-arm-virt-gicv2:
```

Comment:
Missing `name: qemu`

The `name` is the actual device type name used when submitting jobs in LAVA, which has to be `qemu`.

Reply:
No, we need to get away from qemu, it shadow too many different machines.
We should now start to use device aliases since LAVA support it

Reply:
This comment answer also the previous comment (which I cannot answer, I dont know why). With device alias, we can distinguished different uefi-boot

Reply:
Yes it's confusing to have `qemu` as a single device type when we're actually running many different things.  However, LAVA aliases are not ideal because they need to be manually configured in each lab, kept the same across labs otherwise jobs won't run, and they are specific to LAVA so other types of labs would probably need to create some kind of hard-coded mapping somewhere.

Instead, we can take advantage of the fact that the LAVA device type name is independent from the device type as known to KernelCI.  See #159 which removes the unnecessary "name" attribute and replaces it with "type_name" which is what is meant to be used when submitting jobs.  The logical KernelCI device name is passed separately in the job meta-data.